lib/ui/views/projects/project_details_view.dart

```diff
@@ -319,10 +325,14 @@ class _ProjectDetailsViewState extends State<ProjectDetailsView> {
                   child: Text('CANCEL'),
                   onPressed: () => Navigator.pop(context),
                 ),
-                FlatButton(
-                  child: Text('ADD'),
-                  onPressed: () => onAddCollaboratorsPressed(context),
-                ),
+                // FlatButton(
+                //   child: Text('ADD'),
+                //   onPressed: () => onAddCollaboratorsPressed(context),
+                // ),
+                AddButton(
+                    addFunction: onAddCollaboratorsPressed,
+                    context: context,
+                    key: addButtonGlobalKey),
```

Comment:
Here as well.

Reply:
Done both changes @manjotsidhu . Actually I was in a bit hurry so i just done required changes on Github itself, hope two separate commits won't create much hassle. 😅